apps/coscrad-frontend/src/components/resources/media-items/media-item-index.presenter.tsx

```diff
@@ -1,11 +1,37 @@
-import { MediaItemIndexState } from '../../../store/slices/resources/media-items/types';
-import { GenericIndexPresenter } from '../../../utils/generic-components/presenters/generic-index-presenter';
-
-export const MediaItemIndexPresenter = (mediaIndexResult: MediaItemIndexState) => {
-    /**
-     *  TODO [https://www.pivotaltracker.com/story/show/183681839]
-     * We may some day read the actions and allow for bulk command execution in
-     * an index view.
-     */
-    return <GenericIndexPresenter {...mediaIndexResult} indexLabel={'Media Items'} />;
+import { IIndexQueryResult, IMediaItemViewModel } from '@coscrad/api-interfaces';
+import { HeadingLabel, IndexTable } from '../../../utils/generic-components/presenters/tables';
+import { CellRenderersDefinition } from '../../../utils/generic-components/presenters/tables/generic-index-table-presenter/types/cell-renderers-definition';
+import { renderAggregateIdCell } from '../utils/render-aggregate-id-cell';
+
+export const MediaItemIndexPresenter = (
+    mediaIndexResult: IIndexQueryResult<IMediaItemViewModel>
+) => {
+    const { data: detailResult } = mediaIndexResult;
+
+    const medias = detailResult.map(({ data }) => data);
+
+    const headingLabels: HeadingLabel<IMediaItemViewModel>[] = [
+        {
+            propertyKey: 'title',
+            headingLabel: 'Title',
+        },
+        {
+            propertyKey: 'titleEnglish',
+            headingLabel: 'English',
+        },
+    ];
+
+    const cellRenderersDefinition: CellRenderersDefinition<IMediaItemViewModel> = {
+        title: renderAggregateIdCell,
```

Comment:
I wouldn't do it this way, mainly because it's inconsistent with other index views. Instead, add an extra column for `id` in the headings definiton and then here use
```ts
  id: renderAggregateIdCell,
```

Reply:
Fixed.